test/index.js

```diff
@@ -1065,6 +1065,102 @@ describe('PubSub', function() {
     });
   });
 
+  describe('request', function() {
```

Comment:
I believe this is meant to read `getClient_`

Reply:
sure :)